Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -63,7 +63,10 @@ final class AttributeTests: XCTestCase {
         DiagnosticSpec(message: "expected ':' in attribute argument", fixIts: ["insert ':'"]),
         DiagnosticSpec(message: "expected ')' to end attribute", fixIts: ["insert ')'"]),
         DiagnosticSpec(message: "expected declaration after attribute", fixIts: ["insert declaration"]),
-      ]
+      ],
+      fixedSource: """
+        @_specialize(e:) <#declaration#>
```

Comment:
I would expect us to also complain that there is no value after `:`.

Reply:
https://github.com/apple/swift-syntax/issues/1605